backend/src/test/java/gov/cdc/usds/simplereport/service/ApiUserServiceTest.java

```diff
@@ -159,7 +159,7 @@ void getUser_withAdminUser_withOktaMigrationDisabled_success() {
 
     UserInfo userInfo = _service.getUser(apiUser.getInternalId());
 
-    verify(_dbOrgRoleClaimsService, times(0)).getOrganizationRoleClaims((ApiUser) any());
+    verify(_dbOrgRoleClaimsService, times(1)).getOrganizationRoleClaims((ApiUser) any());
```

Comment:
⚠️ Had to increment by 1 in this test file because of how we are comparing the role claims -- we fetch from the DB regardless of whether the feature flag is on or off.